centinel/primitives/http.py

```diff
@@ -49,10 +52,64 @@ def get_request(host, path="/", headers=None, ssl=False,
     }
     # the external result is used when threading to store
     # the results in the list container provided.
-    if external is not None and type(external) is dict:
-        external[url] = result
+    # if external is not None and type(external) is dict:
+    #    external[url] = result
     return result
 
+
+def get_request(host, path="/", headers=None, ssl=False,
+                external=None, url=None):
+    http_results = {}
+    first_response = _get_http_request(host, path, headers, ssl, external, url)
+    if "failure" in first_response["response"]:  # If there was an error, just ignore redirects and return
+        if external is not None and type(external) is dict:
+            external[url] = first_response
+            return first_response
+    # Checks HTTP Status code and location header to see if the webpage calls for a redirect
+    is_redirecting = str(first_response["response"]["status"]).startswith("3") and "location"\
```

Comment:
I would prefer to use a temp variable for this rather than a line continuation. It will make the code more readable to either make the status code string into a temporary variable or make that the startswith call a boolean and do something like (stat_started_with_3 and ...